zingolib/src/lightclient/test_features.rs

```diff
@@ -1,12 +1,13 @@
 use zcash_client_backend::{PoolType, ShieldedProtocol};
 
 use crate::{
+    data::{proposal::TransferProposal, receivers::transaction_request_from_receivers},
     error::ZingoLibError,
     utils::conversion::{address_from_str, testing::receivers_from_send_inputs},
     wallet::Pool,
 };
 
-use super::*;
+use super::{propose::ProposeSendError, *};
```

Comment:
I am -1 on all `use *` patterns.   Since it was already here, this is not a blocking comment.  Anyone can resolve it.